### PR TITLE
[it] update translations of multiple basics lessons

### PR DESCRIPTION
**Pull request by tajchumber**
Lessons updated:
 - basics/pipe-operator.md
 - basics/modules.md
 - basics/mix.md

### CODE REVIEW

File: en/lessons/basics/mix.md [hunk 1]
```diff
@@ -136,17 +136,17 @@ This is analogous to `bundle install`:
...
 
 ## Environments
 
 Mix, much like Bundler, supports differing environments.
 Out of the box mix works with three environments:
 
-+ `:dev` — The default environment.
-+ `:test` — Used by `mix test`.  Covered further in our next lesson.
-+ `:prod` — Used when we ship our application to production.
+- `:dev` — The default environment.
```
Comment (lex111) [inline en/lessons/basics/mix.md]: Such changes in Markdown probably need to be changed in **all** the translated files?

Comment (tajchumber) [reply en/lessons/basics/mix.md]: @lex111 I don't think it's necessary as this is just something my IDE formatter did automatically.
The result is the same :)

**Comment by tajchumber** [comment]
ping @elixirschool/italian :)

**Comment by doomspork** [comment]
@tajchumber I will merge since it's been almost a month. We typically merge after a few days. In future if you don't get a review in a timely fashion just ping me and we'll get you merged 😁

**Pull request merged**